app/_landing_pages/gateway/secrets-management.yaml

```diff
@@ -138,11 +138,12 @@ rows:
             blocks:
               - type: text
                 text: |
-                  There are two main ways to rotate secrets:
+                  {{site.base_gateway}} can't rotate secrets, it can only retrieve rotated secrets. You must configure secret rotation in your third-party vault settings.
+                  {{site.base_gateway}} can be configured to check for rotated secrets in the following ways:
               - type: unordered_list
                 items:
-                  - Rotate periodically using TTLs (e.g. check for new TLS certificate once per day). This can be configured in the [Vault entity](/gateway/entities/vault/) configuration or you can configure it in your third-party vault's settings.
-                  - Rotate on failure (e.g. on database authentication failure, check if the secrets were updated, and try again). You must write code in {{site.base_gateway}} to configure this, so support is limited to Postgres credentials for now. 
+                  - Check periodically using TTLs (for example, check for new TLS certificate once per day). This can be configured in the [Vault entity](/gateway/entities/vault/) configuration.
```

Comment:
```suggestion
                  - Check periodically using TTLs (for example, check for new TLS certificates once per day). This can be configured in the [Vault entity](/gateway/entities/vault/) configuration.
```